src/mmw/requirements/production.txt

```diff
@@ -1,5 +1,6 @@
 -r base.txt
 
+boto3==1.12.33
```

Comment:
These dependencies need to come prior to `boto` or else we get this error on staging:

```bash
ubuntu@ip-10-0-3-80:~$ envdir /etc/mmw.d/env /opt/app/manage.py shell
Python 2.7.12 (default, Oct  8 2019, 14:14:10)
[GCC 5.4.0 20160609] on linux2
Type "help", "copyright", "credits" or "license" for more information.
(InteractiveConsole)
>>> from django.core.files.storage import default_storage
>>> default_storage.exists('storage_test')
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/django/utils/functional.py", line 238, in inner
    self._setup()
  File "/usr/local/lib/python2.7/dist-packages/django/core/files/storage.py", line 472, in _setup
    self._wrapped = get_storage_class()()
  File "/usr/local/lib/python2.7/dist-packages/storages/backends/s3boto3.py", line 336, in __init__
    self.config = Config(**kwargs)
  File "/usr/local/lib/python2.7/dist-packages/botocore/config.py", line 84, in __init__
    args, kwargs)
  File "/usr/local/lib/python2.7/dist-packages/botocore/config.py", line 109, in _record_user_provided_options
    'Got unexpected keyword argument \'%s\'' % key)
TypeError: Got unexpected keyword argument 'proxies'
>>>
```

The only usage of `boto` I could find was to query the EC2 instance metadata endpoint to append stuff to `ALLOWED_HOSTS`, and we have other solutions for doing this we could copy from a project like OAR. However, I wasn't 100% sure, so I decided to leave it alone.

Reply:
Hm. Yeah, it looks like `boto3` might be depending on a more recent version of `botocore` than  `boto`.

The only other place we use `boto` within the app (outside of instance metadata retrieval) is the `boto_ses_mailer` custom email backend (assembled before `django-amazon-ses` was a thing). Email sending is still working on staging, so I think this change is safe for now.